File: Modul1/C/PRAK104-2310817210003-Muhammad_Azrianzan.c
#include <stdio.h>

int main() {
    int sepatuA = 400000;
    printf("Harga sepatu A adalah %d\n", sepatuA);
    int sepatuB = 350000;
    printf("Harga sepatu B adalah %d\n", sepatuB);
    float diskonSepatuA = 0.13;
    float diskonSepatuB = 0.21;

    int potonganHargaSepatuA = sepatuA * diskonSepatuA;
    int potonganHargaSepatuB = sepatuB * diskonSepatuB;

    int sepatuADiskon = sepatuA - potonganHargaSepatuA;
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", sepatuADiskon);
    int sepatuBDiskon = sepatuB - potonganHargaSepatuB;
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d\n", sepatuBDiskon);

    return 0;
}

File: Modul1/Python/PRAK104-2310817210003-Muhammad_Azrianzan.py
sepatuA = 400000
print("Harga sepatu A adalah", sepatuA)
sepatuB = 350000
print("Harga sepatu B adalah", sepatuB)
diskonSepatuA = 0.13
diskonSepatuB = 0.21

potonganHargaSepatuA = sepatuA * diskonSepatuA
potonganHargaSepatuB = sepatuB * diskonSepatuB

sepatuADiskon = int(sepatuA - potonganHargaSepatuA)
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {sepatuADiskon}")
sepatuBDiskon = int(sepatuB - potonganHargaSepatuB)
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {sepatuBDiskon}")